developer_manual/app/backgroundjobs.rst

```diff
@@ -4,36 +4,72 @@ Background Jobs (Cron)
 
 .. sectionauthor:: Bernhard Posselt <dev@bernhard-posselt.com>
 
-Background/cron jobs are usually registered in the :file:`appinfo/app.php` by using the **addRegularTask** method, the class and the method to run:
+ownCloud supports background job functionality (otherwise known as `Cron jobs`_).
+To create them requires two steps to be completed:
 
-.. code-block:: php
+- Create a job class
+- Register the class with ownCloud
 
-    <?php
-    \OCP\Backgroundjob::addRegularTask('\OCA\MyApp\Cron\SomeTask', 'run');
+Create a job class
+==================
 
-The class for the above example would live in :file:`cron/sometask.php`. Try to keep the method as small as possible because its hard to test static methods. Simply reuse the app container and execute a service that was registered in it.
+The first step is to create a job class, which will provide the job functionality. 
+For this example, we will call it: :file:`lib/Cron/SomeTask.php`. 
+The class only needs to define one, static, method called ``run``. 
+In this example, we’re retrieving a service from the container, and in turn
+calling its ``run`` method.
 
-.. code-block:: php
+.. NOTE:: 
+   Try to keep the method as small as possible, because its hard to test static
+   methods.
 
-    <?php
-    namespace OCA\MyApp\Cron;
+.. literalinclude:: ../examples/cron/SomeTask.php
+   :language: php
+   :linenos:
 
-    use \OCA\MyApp\AppInfo\Application;
+Register the class with ownCloud
+================================
 
-    class SomeTask {
+Next, you need to register the job as a background job. 
+This is done in :file:`appinfo/info.xml` by adding a job element, containing the
+name of the job class, to the ``background-jobs`` element. 
+The example below shows how to add the ``SomeTask`` class, which we just
+created, as a background job.:
 
-        public static function run() {
-            $app = new Application();
-            $container = $app->getContainer();
-            $container->query('SomeService')->run();
-        }
+.. code-block:: xml
+   
+    <background-jobs>
+        <job>\OCA\MyApp\Cron\SomeTask</job>
+    </background-jobs>
 
-    }
+Testing
+=======
 
-Dont forget to configure the cron service on the server by executing::
+To test the job classes, you can run Cron manually, as in the example below:
 
+    sudo -u http php cron.php
+
+After doing so, you will need to reset the job to allow it to be run, manually, again. 
+To do this, go to the database and run the following SQL query:
+
+.. code-block: sql
+
+    UPDATE oc_jobs SET last_run=0,last_checked=0,reserved_at=0;
+
+Is The Cron Service Running?
+============================
+
+Finally, don’t forget to add the ownCloud Cron process in the web server’s `crontab`_. To do this, first open the web server’s crontab for editing by running::
```

Comment:
That section is a slightly reworked version of what was there. Happy to remove it and, instead, link to the relevant documentation.